src/pt/lermangaonline/src/eu/kanade/tachiyomi/extension/pt/lermangaonline/LerMangaOnline.kt

```diff
@@ -0,0 +1,130 @@
+package eu.kanade.tachiyomi.extension.pt.lermangaonline
+
+import eu.kanade.tachiyomi.extension.pt.lermangaonline.LerMangaOnlineFilters.GenresFilter
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.concurrent.TimeUnit
+
+class LerMangaOnline : ParsedHttpSource() {
+    override val name = "Ler Mangá Online"
+
+    override val baseUrl = "https://lermangaonline.com.br"
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client: OkHttpClient = network.cloudflareClient.newBuilder()
+        .rateLimitHost(baseUrl.toHttpUrl(), 1, 1, TimeUnit.SECONDS)
+        .build()
+
+    private fun String.toChapterName() =
+        trim().replace("\\d{1,2}-\\d{1,2}-\\d{2,4}".toRegex(), "")
+
+    private fun String.toChapterNumber(): Float =
+        "\\d+".toRegex().find(trim())?.value?.toFloat() ?: 0F
+
+    override fun chapterFromElement(element: Element) = SChapter.create().apply {
+        name = element.selectFirst("div.capitulo")!!.text().toChapterName()
```

Comment:
```suggestion
        name = element.selectFirst("div.capitulo")!!.ownText()
```

You can use [`Element.ownText()`](https://jsoup.org/apidocs/org/jsoup/nodes/Element.html#ownText()) to only get the text inside the element and not any of its children.

Reply:
Fix: https://github.com/keiyoushi/extensions-source/pull/657/commits/a5aba2f43d93ad58f9eec6a131b700f3e1da23ad